zk-sdk-wasm-js/src/proof_data/percentage_with_cap.rs

```diff
@@ -1,42 +1,39 @@
 use {
-    crate::encryption::pedersen::{WasmPedersenCommitment, WasmPedersenOpening},
+    crate::encryption::pedersen::{PedersenCommitment, PedersenOpening},
     js_sys::Uint8Array,
-    solana_zk_sdk::zk_elgamal_proof_program::proof_data::{
-        percentage_with_cap::{PercentageWithCapProofContext, PercentageWithCapProofData},
-        ZkProofData,
-    },
+    solana_zk_sdk::zk_elgamal_proof_program::proof_data::{percentage_with_cap, ZkProofData},
     wasm_bindgen::prelude::*,
 };
 
 /// A percentage-with-cap proof. This proof is used to certify that a transfer
 /// amount is within a certain percentage of a base amount, with a cap.
-#[wasm_bindgen(js_name = "PercentageWithCapProof")]
-pub struct WasmPercentageWithCapProofData {
-    pub(crate) inner: PercentageWithCapProofData,
```

Comment:
I like that all of the types now match the inner type name, with the `*Data` suffix